File: SistemaDecomisos/web/js/objects/Funcionario.js
function Funcionario(identificacion, puesto, nombre, apellido1, apellido2, idFuncionario) {
    this.Funcionario(identificacion, puesto, nombre, apellido1, apellido2, idFuncionario);
}

Funcionario.prototype = {
    identificacion: "",
    puesto: "",
    nombre: "",
    apellido1: "",
    apellido2: "",
    idFuncionario: 0,
    Funcionario: function (identificacion, puesto, nombre, apellido1, apellido2, idFuncionario) {
        this.identificacion = identificacion;
        this.puesto = puesto;
        this.nombre = nombre;
        this.apellido1 = apellido1;
        this.apellido2 = apellido2;
        this.idFuncionario = idFuncionario;
    }
};

Funcionario.from = function (plain) {
    funcionario = new Funcionario(
            plain.identificacion,
            plain.puesto,
            plain.nombre,
            plain.apellido1,
            plain.apellido2,
            plain.idFuncionario);
    return funcionario;
};

Funcionario.to = function (funcionario) {
    return {
        _class: 'Funcionario',
        identificacion: funcionario.idFuncionario,
        puesto: funcionario.puesto,
        nombre: funcionario.nombre,
        apellido1: funcionario.apellido1,
        apellido2: funcionario.apellido2,
        idFuncionario: funcionario.idFuncionario
    };
};
 

File: SistemaDecomisos/src/java/model/Funcionario.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package model;

import java.io.Serializable;

/**
 *
 * @author Marco
 */
public class Funcionario extends Persona implements Serializable, Jsonable{
    private int idFuncionario;
    private String puesto;

    public Funcionario() {
    }

    public Funcionario(int idFuncionario,String puesto, String identificacion, String nombre, String apellido1, String apellido2) {
        super(identificacion, nombre, apellido1, apellido2);
        this.idFuncionario = idFuncionario;
        this.puesto=puesto;
    }

    public int getIdFuncionario() {
        return idFuncionario;
    }

    public void setIdFuncionario(int idFuncionario) {
        this.idFuncionario = idFuncionario;
    }

    public String getPuesto() {
        return puesto;
    }

    public void setPuesto(String puesto) {
        this.puesto = puesto;
    }

    
    
}
